.gitignore

```diff
@@ -4,5 +4,5 @@ config.js
 npm-debug.log*
 TODO.txt
 dest/
-.env
```

Comment:
IIRC having .env in gitignore is a security thing- prevents us from accidentally committing keys. What are you trying to do here?